cmd/copy.go

```diff
@@ -1105,7 +1109,9 @@ func (cca *cookedCopyCmdArgs) launchFollowup(priorJobExitCode common.ExitCode) {
 		glcm.AllowReinitiateProgressReporting()
 		cca.followupJobArgs.priorJobExitCode = &priorJobExitCode
 		err := cca.followupJobArgs.process()
-		if err != nil {
+		if err == NothingToRemoveError {
+			glcm.Error("Cleanup completed (nothing needed to be deleted)")
```

Comment:
Did you mean to call glcm.Error? It'll have a non-zero exit code. If it's intended to be handled gracefully, Exit is fine right?

Reply:
Good point, thanks. I'll fix it now.